linked_list/linked_list.cpp

```diff
@@ -0,0 +1,325 @@
+/*
+* C++ Program to Implement Linked List
+*/
+#include <iostream>
+#include <cstdio>
+#include <cstdlib>
+
+using namespace std;
+/*
+* Node Declaration
+*/
+struct node
+{
+	int info;
+	struct node *next;
+}*start;
+
```

Comment:
Use class instead of structs. (https://google.github.io/styleguide/cppguide.html#Structs_vs._Classes)
Please change your coding style for this PR. Curly braces should look like this.

```
class LinkedList {
    /*
         code
    */
}

```

Please make changes to all function.